(DOCSP-11550) Search for documents with a search template (#29)
mongodb/docs-mongodb-vscode: source/includes/admonitions/document-search-template.rst

.. note::

   You can open a :ref:`JavaScript Playground <vsce-crud>` 

>    pre-configured to search your deployment by hovering over the 

Reviewer: ```suggestion
   pre-configured to search a collection by hovering over the 
```
I'd suggest replacing `your deployment` with `a collection` as the template contains a find for the `database.collection` where you were in the tree when you clicked on the icon.
